.travis.yml

```diff
@@ -1,62 +1,54 @@
 language: python
 python:
-  - 2.7_with_system_site_packages
-  - 3.5.1
-  - 3.6.2
-  - pypy
+  - 2.7
+  - 3.5
+  - 3.6
 cache:
+  pip: true
+  apt: true
   directories:
-    - $HOME/.cache/pip
+    - /tmp/ffmpeg/bin
 sudo: false
```

Comment:
My take is you don't need the sudo and dist lines because these are not Travis defaults and it would be a bummer to lock in trusty when Travis updates distro but this is minor stuff.